### ISSUE TITLE
renamepackage.sh doesn't work on Windows under MingGW bash

### ISSUE THREAD

**Issue by stuartsoft** [madebyatomicrobot/android-starter-project#17]
It would be nice to have a solution for using renamepackage.sh on Windows. Currently does not work properly under MinGW Bash.

**Comment by patrickhammond** [comment]
What CLI commands are missing?

**Comment by stuartsoft** [comment]
```
stuart@DESKTOP-BNDOSEJ MINGW64 ~/testproject (master)
$ ./renamePackage.sh com.foo.bar
rm: cannot remove '*.iml': No such file or directory
rm: cannot remove 'local.properties': No such file or directory
gawk: cmd. line:1: (FILENAME=- FNR=1) warning: gensub: third argument `./app/src/androidTest/java/com/mycompany/myapp/data/api/github/model/CommitTests.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=2) warning: gensub: third argument `./app/src/androidTest/java/com/mycompany/myapp/EspressoTestRule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=3) warning: gensub: third argument `./app/src/androidTest/java/com/mycompany/myapp/MainApplicationDaggerMockRule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=4) warning: gensub: third argument `./app/src/androidTest/java/com/mycompany/myapp/RecyclerViewMatcher.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=5) warning: gensub: third argument `./app/src/androidTest/java/com/mycompany/myapp/TestUtils.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=6) warning: gensub: third argument `./app/src/androidTest/java/com/mycompany/myapp/ui/devsettings/DevSettingsActivityEspressoTests.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=7) warning: gensub: third argument `./app/src/androidTest/java/com/mycompany/myapp/ui/main/MainActivityEspressoTest.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=8) warning: gensub: third argument `./app/src/debug/java/com/mycompany/myapp/app/BuildConfigApplicationInitialization.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=9) warning: gensub: third argument `./app/src/debug/java/com/mycompany/myapp/app/RobolectricApplication.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=10) warning: gensub: third argument `./app/src/debug/java/com/mycompany/myapp/modules/CrashReporterModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=11) warning: gensub: third argument `./app/src/debug/java/com/mycompany/myapp/monitoring/LoggerModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=12) warning: gensub: third argument `./app/src/debug/java/com/mycompany/myapp/ui/RiseAndShine.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=13) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/app/SSLDevelopmentHelper.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=14) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/app/VariantApplicationComponent.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=15) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/app/VariantModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=16) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/app/VariantSettings.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=17) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/ui/devsettings/DevSettingsActivity.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=18) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/ui/devsettings/DevSettingsComponent.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=19) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/ui/devsettings/DevSettingsFragment.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=20) warning: gensub: third argument `./app/src/dev/java/com/mycompany/myapp/ui/devsettings/DevSettingsPresenter.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=21) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/app/AndroidModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=22) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/app/ApplicationComponent.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=23) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/app/AppModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=24) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/app/HasComponent.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=25) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/app/MainApplication.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=26) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/app/Settings.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=27) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/data/api/Api.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=28) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/data/api/github/GitHubApiService.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=29) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/data/api/github/GitHubService.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=30) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/data/api/github/model/Commit.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=31) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/data/DataModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=32) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/monitoring/CrashlyticsCrashReporter.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=33) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/monitoring/CrashReporter.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=34) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/monitoring/LoggingOnlyCrashReporter.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=35) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/ui/ActivityScope.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=36) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/ui/main/MainActivity.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=37) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/ui/main/MainComponent.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=38) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/ui/main/MainFragment.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=39) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/ui/main/MainPresenter.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=40) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/ui/Resource.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=41) warning: gensub: third argument `./app/src/main/java/com/mycompany/myapp/util/RxUtils.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=42) warning: gensub: third argument `./app/src/prod/java/com/mycompany/myapp/app/VariantApplicationComponent.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=43) warning: gensub: third argument `./app/src/prod/java/com/mycompany/myapp/app/VariantModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=44) warning: gensub: third argument `./app/src/prod/java/com/mycompany/myapp/app/VariantSettings.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=45) warning: gensub: third argument `./app/src/release/java/com/mycompany/myapp/app/BuildConfigApplicationInitialization.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=46) warning: gensub: third argument `./app/src/release/java/com/mycompany/myapp/modules/CrashReporterModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=47) warning: gensub: third argument `./app/src/release/java/com/mycompany/myapp/monitoring/LoggerModule.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=48) warning: gensub: third argument `./app/src/release/java/com/mycompany/myapp/monitoring/model/NoOpTree.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=49) warning: gensub: third argument `./app/src/test/java/com/mycompany/myapp/SimpleRobolectricTest.java' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=50) warning: gensub: third argument `./app/src/test/java/com/mycompany/myapp/SimpleTests.java' treated as 1
sed: can't read s/com.mycompany.myapp/com.foo.bar/g: No such file or directory
gawk: cmd. line:1: (FILENAME=- FNR=1) warning: gensub: third argument `./app/lint.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=2) warning: gensub: third argument `./app/src/debug/AndroidManifest.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=3) warning: gensub: third argument `./app/src/debug/res/values/strings.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=4) warning: gensub: third argument `./app/src/dev/AndroidManifest.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=5) warning: gensub: third argument `./app/src/dev/res/layout/activity_dev_settings.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=6) warning: gensub: third argument `./app/src/dev/res/layout/content_dev_settings.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=7) warning: gensub: third argument `./app/src/dev/res/layout/fragment_dev_settings.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=8) warning: gensub: third argument `./app/src/dev/res/values/strings.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=9) warning: gensub: third argument `./app/src/main/AndroidManifest.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=10) warning: gensub: third argument `./app/src/main/res/layout/activity_main.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=11) warning: gensub: third argument `./app/src/main/res/layout/content_main.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=12) warning: gensub: third argument `./app/src/main/res/layout/fragment_main.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=13) warning: gensub: third argument `./app/src/main/res/layout/row_commit_summary.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=14) warning: gensub: third argument `./app/src/main/res/values/strings.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=15) warning: gensub: third argument `./app/src/main/res/values/styles.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=16) warning: gensub: third argument `./app/src/main/res/values-v21/styles.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=17) warning: gensub: third argument `./app/src/prod/AndroidManifest.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=18) warning: gensub: third argument `./app/src/release/res/values/strings.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=19) warning: gensub: third argument `./config/checkstyle/checkstyle.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=20) warning: gensub: third argument `./config/findbugs/exclude.xml' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=21) warning: gensub: third argument `./config/pmd/pmd-ruleset.xml' treated as 1
sed: can't read s/com.mycompany.myapp/com.foo.bar/g: No such file or directory
gawk: cmd. line:1: (FILENAME=- FNR=1) warning: gensub: third argument `./app/src/test/resources/robolectric.properties' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=2) warning: gensub: third argument `./gradle/wrapper/gradle-wrapper.properties' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=3) warning: gensub: third argument `./gradle.properties' treated as 1
sed: can't read s/com.mycompany.myapp/com.foo.bar/g: No such file or directory
gawk: cmd. line:1: (FILENAME=- FNR=1) warning: gensub: third argument `./app/build.gradle' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=2) warning: gensub: third argument `./app/generator.gradle' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=3) warning: gensub: third argument `./build.gradle' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=4) warning: gensub: third argument `./distribution/keys/sample.gradle' treated as 1
gawk: cmd. line:1: (FILENAME=- FNR=5) warning: gensub: third argument `./settings.gradle' treated as 1
mv: './build.gradle' and './build.gradle' are the same file
sed: can't read s/com.mycompany.myapp/com.foo.bar/g: No such file or directory
Initialized empty Git repository in C:/Users/stuart/testproject/.git/
warning: LF will be replaced by CRLF in app/build.gradle.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/generator.gradle.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/lint.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/androidTest/java/com/foo/bar/EspressoTestRule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/androidTest/java/com/foo/bar/MainApplicationDaggerMockRule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/androidTest/java/com/foo/bar/RecyclerViewMatcher.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/androidTest/java/com/foo/bar/TestUtils.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/androidTest/java/com/foo/bar/data/api/github/model/CommitTests.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/androidTest/java/com/foo/bar/ui/devsettings/DevSettingsActivityEspressoTests.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/androidTest/java/com/foo/bar/ui/main/MainActivityEspressoTest.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/debug/AndroidManifest.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/debug/java/com/foo/bar/app/BuildConfigApplicationInitialization.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/debug/java/com/foo/bar/app/RobolectricApplication.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/debug/java/com/foo/bar/modules/CrashReporterModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/debug/java/com/foo/bar/monitoring/LoggerModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/debug/java/com/foo/bar/ui/RiseAndShine.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/debug/res/values/strings.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/AndroidManifest.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/app/SSLDevelopmentHelper.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/app/VariantApplicationComponent.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/app/VariantModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/app/VariantSettings.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/ui/devsettings/DevSettingsActivity.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/ui/devsettings/DevSettingsComponent.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/ui/devsettings/DevSettingsFragment.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/java/com/foo/bar/ui/devsettings/DevSettingsPresenter.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/res/layout/activity_dev_settings.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/res/layout/content_dev_settings.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/res/layout/fragment_dev_settings.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/dev/res/values/strings.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/AndroidManifest.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/app/AndroidModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/app/AppModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/app/ApplicationComponent.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/app/HasComponent.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/app/MainApplication.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/app/Settings.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/data/DataModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/data/api/Api.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/data/api/github/GitHubApiService.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/data/api/github/GitHubService.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/data/api/github/model/Commit.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/monitoring/CrashReporter.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/monitoring/CrashlyticsCrashReporter.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/monitoring/LoggingOnlyCrashReporter.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/ui/ActivityScope.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/ui/Resource.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/ui/main/MainActivity.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/ui/main/MainComponent.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/ui/main/MainFragment.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/ui/main/MainPresenter.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/java/com/foo/bar/util/RxUtils.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/res/layout/activity_main.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/res/layout/content_main.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/res/layout/fragment_main.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/res/layout/row_commit_summary.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/res/values-v21/styles.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/res/values/strings.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/main/res/values/styles.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/prod/AndroidManifest.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/prod/java/com/foo/bar/app/VariantApplicationComponent.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/prod/java/com/foo/bar/app/VariantModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/prod/java/com/foo/bar/app/VariantSettings.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/release/java/com/foo/bar/app/BuildConfigApplicationInitialization.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/release/java/com/foo/bar/modules/CrashReporterModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/release/java/com/foo/bar/monitoring/LoggerModule.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/release/java/com/foo/bar/monitoring/model/NoOpTree.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/release/res/values/strings.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/test/java/com/foo/bar/SimpleRobolectricTest.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/test/java/com/foo/bar/SimpleTests.java.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/src/test/resources/robolectric.properties.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in build.gradle.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in config/checkstyle/checkstyle.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in config/findbugs/exclude.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in config/pmd/pmd-ruleset.xml.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in distribution/keys/sample.gradle.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in gradle.properties.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in gradle/wrapper/gradle-wrapper.properties.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in settings.gradle.
The file will have its original line endings in your working directory.
```

**Comment by stuartsoft** [comment]
Not immediately sure what's wrong/missing. MingGW should have `gawk` `rm` `find` etc.
It seems to be generating quite a few warnings though when it runs and when I open it in Android Studio all .java classes are still using the old package name

**Comment by stuartsoft** [comment]
https://github.com/madebyatomicrobot/android-starter-project/pull/18

**Comment by stuartsoft** [comment]
Resolved in PR #18
